packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -165,31 +94,65 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
   const context = useFluentContext();
   const { setStart, setEnd } = useTelemetry(Datepicker.displayName, context.telemetry);
   setStart();
+  const dateSelectionErrorString = 'A date selection is required';
   const datepickerRef = React.useRef<HTMLElement>();
-  const [open, setOpen] = React.useState<boolean>(false);
+  const [openState, setOpenState] = useAutoControlled<OpenState>({
+    defaultValue: props.defaultCalendarOpenState ? OpenState.Open : OpenState.Closed,
+    value: undefined,
+    initialValue: OpenState.Closed,
+  });
   const [selectedDate, setSelectedDate] = React.useState<Date | undefined>();
-  const valueFormatter = date => (date ? formatMonthDayYear(date, DEFAULT_LOCALIZED_STRINGS) : '');
-  const {
-    firstDayOfWeek,
-    firstWeekOfYear,
-    dateRangeType,
-    calendar,
-    popup,
-    className,
-    design,
-    styles,
-    variables,
-  } = props;
-  const calendarOptions: IDayGridOptions = {
-    selectedDate: selectedDate ?? props.today ?? new Date(),
-    navigatedDate: selectedDate ?? props.today ?? new Date(),
-    firstDayOfWeek,
-    firstWeekOfYear,
-    dateRangeType,
-  };
+  const [formattedDate, setFormattedDate] = React.useState<string>('');
+  const [error, setError] = React.useState<string>(() =>
+    props.isRequired && !selectedDate ? dateSelectionErrorString : '',
```

Comment:
```suggestion
    props.required && !selectedDate ? dateSelectionErrorString : '',
```

We don't use `is` prefix

Reply:
Fixed.